models/farmer.js

```diff
@@ -6,6 +6,10 @@ const personalInfo = {
     enum: ['Miss', 'Mrs', 'Mr', 'Chief'],
     required: false
   },
+  image_url: {
+    type: String,
+    required: true
```

Comment:
would not make it `required` field in case they don't want to/can't upload the image